### PR TITLE
docs: Added autoland.md page

### LINKED ISSUE #394: Create autoland advanced guide
See docs thread for discusssion. 

PR: #390

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary
Adding of Autoland Guide to describe how to use it and also give real world information about this feature.

### Location
https://docs-git-fork-frankkopp-adv-autoland-guide-flybywire.vercel.app/pilots-corner/advanced-guides/flight-guidance/autoland

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

File: docs/pilots-corner/advanced-guides/flight-guidance/autoland.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+enforced and this may even extend to CAT I runways. 
+
+Autoland can also be subject to local procedures and authorities.
+
+### When Not to Use Autoland
+
+- ATC restrictions
+- Airport capabilities missing or restrictions in place.
+- Aircraft capabilities missing and limitations exceeded.
+- If CAT1 is displayed on the FMA, Autoland is not authorized.
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/flight-guidance/autoland.md:49]:
```suggestion
- If CAT I is displayed on the FMA, Autoland is not authorized.
```

File: docs/pilots-corner/advanced-guides/flight-guidance/autoland.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+All Airbus aircraft are certified to land automatically. However, limitations and conditions specified in the FCOM 
+must be taken into account. Be aware that other not-so-obvious Autoland-limitations, such as maximum airfield 
+altitude, maximum (minimum) GS angle or maximum runway slope, must also be considered. 
+
+In addition, the flight crew must monitor possible day-to-day technical restrictions (stated in the MEL), or the 
+consequence(s) of a failure that may have occurred during the flight and that may downgrade landing capability.
+
+Autoland many not be performed outside of these limitations:
+
+- Max. Crosswind: 20ft
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/flight-guidance/autoland.md]:
```suggestion
- Max. Crosswind: 20kt
```

File: docs/pilots-corner/advanced-guides/flight-guidance/autoland.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+- Aircraft weight above the maximum landing weight
+- Automatic landing is not allowed below -2.000 ft pressure altitude
+
+Minima: 
+
+- ILS CAT II: MDH 100ft
+- ILS CAT III SINGLE: MDH 50ft
+- ILS CAT III DUAL: no DH (minimum RVR 75m)
+
+## Autoland HowTo
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/flight-guidance/autoland.md]:
```suggestion
## Autoland How To
```

File: docs/pilots-corner/advanced-guides/flight-guidance/autoland.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+- At 40ft: the FMA must show FLARE or otherwise the flight crew needs to either perform a manual landing if 
+  visual references are acquired or perform a go around.
+- Autothrust FMA should switch to THRUST IDLE.
+- Between about 20ft and 10ft the aircraft corrects its crosswind crab angle to bring its longitudinal axis in line 
+  with the runway.
+- At 10ft: callout "Retard" commands the PF to set the throttle levers to idle.
+- At touchdown:
+    - FMA shows ROLL OUT - autopilot keeps aircraft on center line
+    - Both thrust levers to reverse thrust (idle or max as appropriate)
+    - PM checks and calls out: "touchdown", "spoilers", "reversers", "auto brake", "descel".
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/flight-guidance/autoland.md]:
```suggestion
    - PM checks and calls out: "touchdown", "spoilers", "reversers", "auto brake", "decel".
```

**Review by Valastiri — COMMENTED** [review]
Initial pre-pilot verification grammar review

File: docs/pilots-corner/advanced-guides/flight-guidance/autoland.md (hunk 1 above)
Comment (frankkopp) [reply docs/pilots-corner/advanced-guides/flight-guidance/autoland.md:49]: It's actually showing CAT1 :)

File: docs/pilots-corner/advanced-guides/flight-guidance/autoland.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+
+- Perform a **go-around** in case of:
+    - AP loss (cavalry charge)                
+    - Amber caution (single chime)            
+    - [Autoland Warning Light](#autoland-warning-light) (below 200’ RA)  
+    - Triple click (downgrading of capability)
+- Proceed in case of:
+    - Fire (red warning: Engine or APU)       
+    - Smoke (red warning: Avionics, Cargo, Lavatory)
+    - Smokes/Fumes in the cockpit/Cabin
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/flight-guidance/autoland.md]:
```suggestion
    - Smokes/Fumes in the Cockpit/Cabin
```
Or cockpit/cabin -- just capilization check

**Review by Valastiri — APPROVED** [review]
Small tidbit otherwise LGTM

**Comment by Valastiri** [comment]
check build

**Pull request merged**